### PR TITLE
Added new Javascript quiz questions #14

### PR DESCRIPTION
**Pull request by arindamkeswani**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/full-quiz.js [hunk 1]
```diff
@@ -1,5 +1,60 @@
 const fullQuiz = [
   {
+    Question:
+      "Which of the following is the default separator in join() method in javascript?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:4]: I think we should add the word "the" between in and join().
Also, please make sure to property capitalize JavaScript.

New sentence: Which of the following is the default separator in the join() method in JavaScript?

File: src/data/full-quiz.js [hunk 2]
```diff
@@ -1,5 +1,60 @@
...
+    Distractor1: "map()",
+    Distractor2: "filter()",
+    Distractor3: "forEach()",
+    Explanation:
+      "The reduce() method iterates over the entire array and returns the value that results from running the “reducer” callback function to completion. The returned result is a single value.",
+    Link: "https://www.freecodecamp.org/news/the-ultimate-guide-to-javascript-array-methods-reduce/",
+  }
+  ,{
+    Question:
+      "Which method iterates over the elements of an array and creates a new array of elements that fulfil the condition(s) specified inside a function within said method in javascript?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:37]: Please properly capitalize JavaScript.

File: src/data/full-quiz.js [hunk 3]
```diff
@@ -1,5 +1,60 @@
...
+    Distractor1: "map()",
+    Distractor2: "reduce()",
+    Distractor3: "forEach()",
+    Explanation:
+      "The filter() method creates a new array and returns all of the items which pass the condition specified in the callback.",
+    Link: "https://www.freecodecamp.org/news/javascript-array-filter-tutorial-how-to-iterate-through-elements-in-an-array/",
+  }
+  ,{
+    Question:
+      "Which method iterates over the elements of an array and creates a new array based on the results of the function specified within said method in javascript?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:48]: Please properly capitalize JavaScript.

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Hi @arindamkeswani ! 

Thank you for your quiz questions. Please review my comments and make those necessary changes. 
Then we can merge your PR.
Thanks!

**Comment by arindamkeswani** [comment]
Made the necessary changes!

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Comment by jdwilkin4** [comment]
HI @arindamkeswani ! 

Congrats on your first PR! 
We look forward to future contributions.

**Pull request merged**